packages/ui/src/views/unified-pipeline-studio/components/unified-pipeline-studio-layout.tsx

```diff
@@ -7,7 +7,7 @@ export const HEADER_HEIGHT = 55
 
 const PipelineStudioLayout = {
   Root: ({ children, className }: { children: ReactNode; className?: string }) => {
-    return <div className={cn('flex grow flex-col', className)}>{children}</div>
+    return <div className={cn('flex grow flex-col min-h-[calc(100vh-55px-45px)]', className)}>{children}</div>
```

Comment:
It would be ideal to find a robust solution that avoids the use of magic numbers. Relying on them throughout the application could make maintenance challenging in the long run.

cc: @abhinavrastogi-harness @knagurski

Reply:
Well, I'm not the one who introduces this, but agree - it can be improved✌️

* related to the [thread above](https://github.com/harness/canary/pull/1367#discussion_r2033218317)

Reply:
Yes, magic numbers are all over our application. We should find a robust solution for it. May be we can add all the sizing values in tailwindcss config and update arbitrary values eslint as error.

Reply:
I agree, the most straightforward option is to add those two sizes to CSS variables with descriptive names.

Ridding the system of magic numbers is one for another PR, but we can at least do these two in this one.

Reply:
Yeah, seems like a reasonable approach. We might want to be defensive and add a prefix to the names incase we collide with something in Harness SaaS. Maybe `--canary-breadcrumbs-height`?

Reply:
I'll add `cn-` prefix for consistency